birdhouse/components/proxy/docker-compose-extra.yml

```diff
@@ -12,12 +12,10 @@ services:
     image: ${PROXY_IMAGE}
     container_name: proxy
     ports:
-      - "80:80"
-      - "443:${PROXY_SECURE_PORT}"
+      - "80:${PROXY_HTTP_PORT}"
```

Comment:
I've been searching the diff in this PR, is `PROXY_HTTP_PORT` configurable or always hardcoded to 80?  I only see it hardcoded.

If always hardcoded, might as well add a comment here that his variable is not configurable, to not give false impression to other.

Reply:
Oh, going over the PR again, `PROXY_HTTP_PORT` is not new.  It is just a rename of `PROXY_SECURE_PORT` which is also hardcoded and not configurable and is a trick to force people to use the wrapper `pavics-compose.sh` and not call `docker-compose directly.

Forget my previous comment.